scripts/system_monitoring_functions/disk_stats.sh

```
@@ -0,0 +1,55 @@
+get_disk_io_stats() {
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)